tests/basictests/dsp-operator.sh

```diff
@@ -59,8 +59,8 @@ function test_metrics() {
 function fetch_runs() {
     header "Fetch the dsp route and verify it works"
 
-    ROUTE=$(oc get route -n ${DSPAPROJECT}  ds-pipeline-ui-sample --template={{.spec.host}})
-    SA_TOKEN=$(oc create token ds-pipeline-ui-sample -n ${DSPAPROJECT})
+    ROUTE=$(oc get route -n ${DSPAPROJECT}  ds-pipeline-sample --template={{.spec.host}})
+    SA_TOKEN=$(oc create token ds-pipeline-sample -n ${DSPAPROJECT})
```

Comment:
For this to work,
we need to include:
https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/internal/mlpipelines-ui/role.yaml.tmpl#L62-L67
in https://github.com/opendatahub-io/data-science-pipelines-operator/blob/main/config/internal/apiserver/role_ds-pipeline.yaml.tmpl
as well.
because the oauth proxy looks for that role https://github.com/opendatahub-io/data-science-pipelines-operator/blob/85ffb2dea9ea3c03e81b539639e4bb2a0cfcfc7b/config/internal/apiserver/deployment.yaml.tmpl#L31